frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts

```diff
@@ -190,17 +190,29 @@ export const sessionRecordingDataLogic = kea<sessionRecordingDataLogicType>([
             },
         ],
     })),
-    listeners(({ values, props, actions, cache }) => ({
-        loadEntireRecording: () => {
-            actions.loadRecordingMeta()
-        },
-        loadRecordingMetaSuccess: () => {
+    listeners(({ values, actions, cache, props }) => ({
+        loadRecording: ({ full }) => {
```

Comment:
so `!full` means `only load metadata` ? 

That threw me... Should it be something like `previewOnly` 

Not blocking but 🤷